### ISSUE TITLE
Text alternative does not present same info 

### ISSUE THREAD

**Issue by shobhit-v** [shobhit-v/demo#93]
## Issue Description
**Impact:** Serious
**Issue Summary:** Text alternative does not present same info 
**Issue Description:** The text alternative does not present the same information as the image.

**WCAG Standards:** wcag2a, wcag2aa, wcag21a, wcag21aa
**Test Case Name:** Test web assessment
**Test Run Name:** Test web assessment_07-27-21-12:23
**Checkpoint Group:** 1.1 Text Alternatives
**Issue Type:** Accessibility
**Unit Type:** Page
**Test Unit:** Test page
**Recommended to fix:**
 RULE%20%3A%20%0D%0AThe%20alternative%20text%20for%20informative%20images%20MUST%20be%20accurately%20conveying%20the%20purpose%20of%20the%20image%2C%20and%20the%20author%27s%20intent%20in%20a%20way%20that%20is%20useful%20to%20those%20who%20cannot%20see%20the%20image.%0D%0A%0D%0AHOW%20TO%20FIX%3A%20%0D%0AFix%20this%20issue%20by%20ensuring%20that%20the%20alternative%20text%20for%20the%20informative%20image%20describes%20its%20intent%2C%20purpose%2C%20or%20meaning.%0D%0A%0D%0A%3Cimg%20src%3D%22sunset.jpg%22%20alt%3D%22The%20sun%20setting%20over%20the%20Pacific%20Ocean%2C%20with%20a%20silhouette%20of%20a%20flying%20seagull%20in%20the%20foreground%22%3E%0D%0A%0D%0AREFERENCE%3A%20%0D%0ADeque%20University%3A%20https%3A//dequeuniversity.com/class/images2/alt-text/informative%0D%0AW3C-WAI%20tutorial%3A%20https%3A//www.w3.org/WAI/tutorials/images/%0D%0A%0D%0ABACKGROUND%3A%20%0D%0APeople%20who%20are%20blind%20cannot%20see%20images%20on%20a%20page.%20In%20order%20to%20give%20people%20who%20cannot%20see%20an%20image%20access%20to%20the%20information%20conveyed%20by%20the%20image%2C%20it%20must%20have%20a%20text%20alternative.%20The%20text%20alternative%20must%20describe%20the%20information%20or%20function%20represented%20by%20the%20image.%20Screen%20readers%20can%20then%20use%20the%20alternative%20text%20to%20convey%20that%20information%20to%20the%20screen%20reader%20user.%0D%0A%0D%0AFor%20informative%20images%20-%20images%20that%20convey%20information%20-%20the%20alternative%20text%20must%20communicate%20the%20intent%2C%20purpose%2C%20or%20meaning%20of%20the%20image%20in%20a%20way%20that%20serves%20as%20a%20true%20alternative%20for%20the%20image.
**Digital Asset Type:** Desktop Web
**Release:** g
**Environment:** csv
**Platform:** cvsa
**Assistive technology:** s
**Checkpoint:** Alternative Text (Informative Images) (1.1.1.b)
**Method:** Manual
**Tested On URL:** www.google.com
